README.md

```diff
@@ -1 +1,3 @@
-# RESTful-app
\ No newline at end of file
+# RESTful-app
+
+A simple RESTful-app that returns a JSON response for a GET request with a query string using NestJS.
```

Comment:
This is a good start, however in real world applications we write a complete guide how to start up the application locally, any configurations, assumptions etc.
README is a place to help anyone who reads this repository